Sources/MySQL/MySQLRowParser.swift

```diff
@@ -12,6 +12,8 @@ internal class MySQLRowParser {
         switch headers[i].type {
           case MySQLFieldType.String, MySQLFieldType.VarString:
             result[headers[i].name] = pointerToString(row[i]!)
+          case MySQLFieldType.Blob:            
+            result[headers[i].name] = pointerToString(row[i]!)
```

Comment:
im curious whether this would fail if the BLOB data being represented was purely binary. for now, since MYSQL_TYPE_BLOB works for TEXT field types (https://dev.mysql.com/doc/refman/5.7/en/c-api-data-structures.html) this is working fine